graph-manager-docs/source/federation.mdx

```diff
@@ -123,38 +123,34 @@ server.listen().then(({ url }) => {
 
 > The managed configuration will default to using the `'current'` variant.  When running Apollo Gateway in an environment where outbound traffic to the internet is restricted, consult the [directions for configuring a proxy](https://www.apollographql.com/docs/apollo-server/proxy-configuration/) within Apollo Server.
 
-### Managed configuration
+## Managed configuration
 
-Managed configuration is the concept of allowing the gateway to update dynamically in response to service changes. In principle, it's important to separate the reliability of your graph from the reliability of your services. For that reason, we recommend running federation in production using a managed configuration, where the gateway picks up config changes not from introspection, but from a set of files owned by your graph describing its current state.
+With **managed configuration**, your GraphQL gateway updates dynamically in response to schema changes in your implementing services. You make these service changes with the `apollo service:push` CLI command, which pushes your service configuration files to Graph Manager and stores them securely.
```

Comment:
Should we reiterate that you need to pass `--serviceName` to the `apollo service:push` command?

Reply:
Including a codeblock here that includes the flag. Really what we need is a definitive command reference to link to in all of these cases, which is a short-to-mid-term goal

Reply:
ok actually linking elsewhere in the article where an applicable codeblock is already shown